source/read/specify-documents-to-return.txt

```diff
@@ -0,0 +1,198 @@
+      Restaurant(name=44 Sw Ristorante & Bar, borough=Manhattan, cuisine=Italian)
+      Restaurant(name=900 Park, borough=Bronx, cuisine=Italian)
+      Restaurant(name=A Voce, borough=Manhattan, cuisine=Italian)
+      ...
+
+.. _kotlin-sync-skip:
+
+Skip
+----
+
+To skip a specified number of documents before returning your query results,
+call the ``skip()`` method and pass in the number of documents to skip. The
+``skip()`` method ignores the specified number of documents in your query
+results and returns the rest.
+
+The following example returns all documents that have a ``cuisine`` field value
+of ``"Italian"`` and skips the first 10 documents:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/read/specify-documents-to-return.kt
+      :start-after: start-skip
+      :end-before: end-skip
+      :language: kotlin
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      Restaurant(name=San Pietro, borough=Manhattan, cuisine=Italian)
+      Restaurant(name=Manetta's Ristorante, borough=Queens, cuisine=Italian)
+      Restaurant(name=Salvi Restaurant, borough=Brooklyn, cuisine=Italian)
+      Restaurant(name=Tommaso Restaurant, borough=Brooklyn, cuisine=Italian)
+      Restaurant(name=Da Rosina Restaurant, borough=Manhattan, cuisine=Italian)
+      ...
+
+Combine Limit, Sort, and Skip
+-----------------------------
+
+You can combine the ``limit()``, ``sort()``, and ``skip()`` methods in a single
+operation. This allows you to set a maximum number of sorted documents to
+return, skipping a specified number of documents before returning.
+
+The following example returns documents with the ``cuisine`` field value of
+``"Italian"``. The results are sorted in alphabetical order, skipping the first
+10 documents and limiting the results to 5 documents:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/read/specify-documents-to-return.kt
+      :start-after: start-limit-sort-skip
+      :end-before: end-limit-sort-skip
+      :language: kotlin
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      Restaurant(name=Acqua, borough=Manhattan, cuisine=Italian)
+      Restaurant(name=Acqua Restaurant, borough=Manhattan, cuisine=Italian)
+      Restaurant(name=Acqua Santa, borough=Brooklyn, cuisine=Italian)
+      Restaurant(name=Acquista Trattoria, borough=Queens, cuisine=Italian)
+      Restaurant(name=Acquolina Catering, borough=Manhattan, cuisine=Italian)
+
+.. note::
+
+   The order in which you call these methods doesn't change the documents
+   that are returned. The driver automatically reorders the calls to perform the
+   sort and skip operations first, and the limit operation afterward.
+
+Additional Information
+----------------------
+
+.. TODO: For more information about specifying a query, see :ref:`kotlin-sync-specify-query`.
+
+For more information about retrieving documents, see :ref:`kotlin-sync-retrieve`.
```

Comment:
Note for reviewers: The retrieve page here is currently also in review. This PR should be merged in after the retrieve page is merged.

[Link to Retrieve page PR](https://github.com/mongodb/docs-kotlin-sync/pull/6)